docs/3x/PT_StaticQuant.md

```diff
@@ -0,0 +1,104 @@
+        2.1.2 [Specify Quantization Rules](#specify-quantization-rules) \
+        2.1.3 [Model Examples](#model-examples) \
+    2.2 [Static Quantization with PT2E Backend](#static-quantization-with-pt2e-backend) \
+        2.2.1 [Usage Sample with PT2E](#usage-sample-with-pt2e)
+
+
+## Introduction
+
+Post-Training Quantization (PTQ) is a technique used to convert a pre-trained floating-point model to a quantized model. This approach does not require model retraining. Instead, it uses calibration data to determine the optimal quantization parameters. Static quantization involves calibrating both weights and activations during the quantization process. Currently, we support two paths to perform static PTQ: [Intel Extension for PyTorch (IPEX)](https://github.com/intel/intel-extension-for-pytorch) and [PyTorch 2 Export Quantization (PT2E)](https://pytorch.org/tutorials/prototype/pt2e_quant_x86_inductor.html).
+
+## Get Started
+
+### Static Quantization with IPEX Backend
+
+Intel Extension for PyTorch (IPEX) provides optimizations specifically for Intel hardware, improving the performance of PyTorch models through efficient execution on CPUs. IPEX supports PTQ, allowing users to quantize models to lower precision to reduce model size and inference time while maintaining accuracy.
+
+The design philosophy of the quantization interface of Intel(R) Neural Compressor is easy-of-use. It requests user to provide `model`, `calibration function`, and `example inputs`. Those parameters would be used to quantize and tune the model. 
+
+`model` is the framework model location or the framework model object.
+
+`calibration function` is used to determine the appropriate quantization parameters, such as `scale` and `zero-point`, for the model's weights and activations. This process is crucial for minimizing the loss of accuracy that can occur when converting from floating-point to lower-precision format.
+
+IPEX leverages just-in-time (JIT) compilation techniques for optimizing the model. `example inputs` is used to trace the computational graph of the model, enabling various optimizations and transformations that are specific to IPEX. This tracing process captures the operations performed by the model, allowing IPEX to apply quantization optimizations effectively. `example inputs` should be representative of the actual data the model will process to ensure accurate calibration.
+
+
+#### Usage Sample with IPEX
+```python
+import intel_extension_for_pytorch as ipex
+from neural_compressor.torch.quantization import StaticQuantConfig, convert, prepare
+
+quant_config = StaticQuantConfig(act_sym=True, act_algo="minmax")
+prepared_model = prepare(model, quant_config=quant_config, example_inputs=example_inputs)
+run_fn(prepared_model)
+q_model = convert(prepared_model)
+```
+
+> [!IMPORTANT]  
+> To use static quantization with the IPEX backend, please explicitly import IPEX at the beginning of your program.
```

Comment:
@violetch24 Please be aware of this usage behavior change that I have aligned with @ftian1  and @xin3he.